go.mod

```diff
@@ -6,8 +6,7 @@ require (
 	github.com/DiSiqueira/GoTree v1.0.1-0.20180907134536-53a8e837f295
 	github.com/benlaurie/objecthash v0.0.0-20180202135721-d1e3d6079fc1
 	github.com/fatih/color v1.10.0
-	github.com/flyteorg/datacatalog v0.3.0
-	github.com/flyteorg/flyteidl v0.18.15
+	github.com/flyteorg/flyteidl v0.18.17
```

Comment:
I think we are up to 0.18.20 now... can you run `go get -u github.com/flyteorg/flyteidl`